pkg/stanza/testutil/mocks.go

```diff
@@ -120,3 +121,173 @@ func (f *FakeOutput) ExpectNoEntry(t testing.TB, timeout time.Duration) {
 		return
 	}
 }
+
+// Operator is an autogenerated mock type for the Operator type
+type Operator struct {
+	mock.Mock
+}
+
```

Comment:
Why copying this code?

Reply:
I realized when removing mockery that the other generated files were no longer being used. Since this would be the only mocked struct, it seemed appropriate to just include it in `mocks.go`.

That said, probably should just leave it as it was, so will revert this.